tools/NuGet/template-nuget/DynamoVisualProgramming.DynamoSamples.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>DynamoVisualProgramming.DynamoSamples</id>
+    <version>@VERSION@</version>
+    <authors>Autodesk</authors>
+    <owners>Autodesk</owners>
+    <licenseUrl>http://opensource.org/licenses/MIT</licenseUrl>
```

Comment:
I think most dynamo assets are licensed with apache 2.0 license.

Reply:
On other templates, LicenseUrl is still MIT. Which one should I choose?

Reply:
hmm, lets keep it consistent then and we'll decide in the future if they should all be changed over at once. Thanks for checking it out!